src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -13,6 +13,7 @@ import { Jazzbar } from "components/templates/Jazzbar";
 import { PartyMap } from "components/templates/PartyMap";
 import { PlayaRouter } from "components/templates/Playa/Router";
 import { ReactionPage } from "components/templates/ReactionPage";
+import { Embeddable } from "components/templates/Embeddable";
```

Comment:
Fix import sort order

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1243/commits/df631f4be96346b9b8d485322d441cac73156a88